source/how-tos/configure_gpio.rst

```diff
@@ -50,45 +57,57 @@ File**\ ” to the required board specific dlt file
 Changes shall be reflected into the dlt file. Open the corresponding dlt
 file in notepad and check if the value is updated accordingly.
 
+**Step 6.** Now build the SBL and flash it.After bootup GPIO changes shall

```

Comment:
Space after full-stop/period. Also, rephrase a bit - SBL will apply the new settings on subsequent boots.

Reply:
Addressed in latest changes